tests/test_location_pi.py

```diff
@@ -0,0 +1,508 @@
+                         "Picking %d has wrong dest location" % picking_idx)
+        self.assertTrue(picking.has_packages,
+                        "Picking %d does not have packages" % picking_idx)
+        self.assertEqual(len(picking.move_line_ids), 1,
+                         "Picking %d has not single move line" % picking_idx)
+
+    def test07_process_pi_count_moves_valid_request_with_package(self):
+        """ Successfully creates the expected picking """
+        self.create_quant(self.apple.id, self.test_location_01.id, 4,
+                          package_id=self.package_one.id)
+        count_moves = [{"package_id": self.package_one.id,
+                        "location_id": self.test_location_01,
+                        "location_dest_id": self.test_location_02.id}]
+
+        pickings = self.test_location_01._process_pi_count_moves(count_moves)
+
+        self.assertEqual(len(pickings), 1,
+                         "Did not create exactly one picking")
+        self._check_picking_loc01_to_loc02(pickings[0], 0)
+        self.assertEqual(pickings[0].move_line_ids[0].package_id.id,
+                         self.package_one.id,
+                         "Picking is not related to the specified package")
+
+    def test08_process_pi_count_moves_valid_request_with_2_packages(self):
+        """
+        Successfully creates the two expected pickings when two
+        count moves items are specified, both with packages.
+        """
+        self.create_quant(self.apple.id, self.test_location_01.id, 4,
+                          package_id=self.package_one.id)
+        self.create_quant(self.banana.id, self.test_location_01.id, 13,
+                          package_id=self.package_two.id)
+        count_moves = [
+            {
+                "package_id": self.package_one.id,
+                "location_id": self.test_location_01,
+                "location_dest_id": self.test_location_02.id
+            },
+            {
+                "package_id": self.package_two.id,
+                "location_id": self.test_location_01,
+                "location_dest_id": self.test_location_02.id
+            }
+        ]
+
+        pickings = self.test_location_01._process_pi_count_moves(count_moves)
+
+        self.assertEqual(len(pickings), 2,
+                         "Did not create exactly two picking")
+
+        for idx in range(2):
+            self._check_picking_loc01_to_loc02(pickings[idx], idx)
+
+        self.assertEqual(pickings[0].move_line_ids[0].package_id.id,
+                         self.package_one.id,
+                         "Picking 0 is not related to the specified package")
+        self.assertEqual(pickings[1].move_line_ids[0].package_id.id,
+                         self.package_two.id,
+                         "Picking 1 is not related to the specified package")
+
+    def test09_process_pi_count_moves_valid_request_with_quant(self):
```

Comment:
Can we have the same test but with 2 quants, different products but in the same package please?